requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 -r requirements.txt
 black==22.10.0
 flake8==5.0.4
-isort==5.10.1
+isort==5.12.0
```

Comment:
Also updated the dev requirements to match the recent fix to the pre-commit hook :slightly_smiling_face: